src/main/java/dinkplugin/domain/PlayerLookupService.java

```diff
@@ -0,0 +1,36 @@
+package dinkplugin.domain;
+
+public enum PlayerLookupService {
+    NONE("None"),
+    OSRS_HISCORE("OSRS HiScore"),
+    CRYSTAL_MATH_LABS("Crystal Math Labs"),
+    TEMPLEOSRS("Temple OSRS"),
+    WISEOLDMAN("Wise Old Man");
+
+    private final String name;
+
+    PlayerLookupService(String name) {
+        this.name = name;
+    }
+
+    public String playerUrl(String playerName) {
```

Comment:
With the links we currently use, we could technically store it as a url prefix but this would make things more complicated if a service we wanted to add support for used a separate syntax (e.g. `"https://forsen.tv/" + playerName + "/hiscore"`) so I think this makes sense